.gitlab/jobs/poodle.yml

```diff
@@ -0,0 +1,60 @@
+##############################################################################
+# Copyright (c) 2016-23, Lawrence Livermore National Security, LLC
+# and RAJA project contributors. See the RAJA/LICENSE file for details.
+#
+# SPDX-License-Identifier: (BSD-3-Clause)
+##############################################################################
+
+########################
+# Overridden shared jobs
+########################
+# We duplicate the shared jobs description and add necessary changes for RAJA.
+# We keep ${PROJECT_<MACHINE>_VARIANTS} and ${PROJECT_<MACHINE>_DEPS} So that
+# the comparison with the original job is easier.
+
+clang_14_0_6:
+  variables:
+    SPEC: " ~shared +openmp +omptask +tests %clang@14.0.6"
+  extends: .job_on_poodle
+
+gcc_10_3_1:
+  variables:
+    SPEC: " ~shared +openmp +omptask +tests %gcc@10.3.1"
+    POODLE_JOB_ALLOC: "--time=60 --nodes=1"
+  extends: .job_on_poodle
+
+intel_19_1_2_gcc_10_3_1:
+  variables:
+    SPEC: " ~shared +openmp +omptask +tests %intel@19.1.2.gcc.10.3.1"
+    POODLE_JOB_ALLOC: "--time=90 --nodes=1"
+  extends: .job_on_poodle
+
+intel_2022_1_0:
+  variables:
+    SPEC: "${PROJECT_POODLE_VARIANTS} %intel@2022.1.0 ${PROJECT_POODLE_DEPS}"
+  allow_failure: true
+  extends: .job_on_poodle
+
+############
+# Extra jobs
+############
+# We do not recommend using ${PROJECT_<MACHINE>_VARIANTS} and
+# ${PROJECT_<MACHINE>_DEPS} in the extra jobs. There is no reason not to fully
+# describe the spec here.
+
+clang_14_0_6_openmp_off:
+  variables:
+    SPEC: " ~shared ~openmp +tests %clang@14.0.6"
```

Comment:
Why is this ~openmp while others are +openmp?

Reply:
Hmm, I wouldn’t be able to track the history of it, but it’s not a typo, the name of the job is coherent.

I would say, I you want it to be "+openmp", we’ll make it that way :D